setup.cfg

```diff
@@ -1,3 +1,4 @@
 [flake8]
 ignore = E265
 exclude=webapp-django/crashstats/*/migrations/*,docs/conf.py,socorro/unittest/testlib/*
+max-line-length = 100
```

Comment:
All of those situations work fine with 100 characters for me, so I'm not sure why you're experiencing something different. Are you using a low resolution monitor?

Also, honestly, the code isn't linted at all currently, so there's a lot of code that already exceeds 100 characters. For example:

https://github.com/mozilla/socorro/blob/master/socorro/database/cachedIdAccess.py

This is mostly a cosmetic change that doesn't affect the status quo at all.